feat(global): #183703810 real css modules

demogar opened the pull request:
<!--
  This is the default template for pull requrests.

  ====================================================================
  Follow this naming convention for your PR:

  chore|feat|feature|bugfix|fix|docs|style|refactor|perf|test (<Pivotal Tracker ID>)?: <Short Description>
-->

# Feature

## Feature Description

- Add real CSS modules
- Add a separate token file for handling the css tokens as variables
- Import two different css files

## Screenshot

![image](https://user-images.githubusercontent.com/94550/201692756-08a347ed-23f7-4cc0-bc84-4866079b4496.png)

## Ticket

[Ticket 183703810](https://www.pivotaltracker.com/story/show/183703810)

## Checklist

- [ ] Tests
- [ ] Validated Against Design

demogar reviewed (approved):
Makes sense

demogar commented on src/index.js:
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+import './normalize.css';
```
however, should it be after the `index.css`?

kvuzet replied:
oh good call, it should be before

kvuzet replied:
fixed

demogar committed:
feat(global): #183703810 add modular css names

add modular css names
remove internal scss variables

demogar committed:
chore(global): #183785957 add workflow for testing components

add workflow for testing components on a new PR and push

demogar committed:
feat(global): #183703810 add real css modules for the entire project

change scss files to be css modules
remove atlas prefix
use css modules for the classnames

demogar committed:
feat(atlas-react): #183703810 change plop configuration for css modules

demogar committed:
feat(atlas-react): #183703810 update snaps and plop

update snaps for basic button
update plop command to prepend the .Atlas class
